### ISSUE TITLE
cmake build error

### ISSUE THREAD

**Issue by TankPi** [MOLAorg/mola#38]
Thanks for this great project. When I try to build the latest commit (b02117627adb465c6afa9ce2ba7c1e670353f541) of the mola code, I get the following error:

![Screenshot from 2021-03-18 14-18-52](https://user-images.githubusercontent.com/11289683/111632569-e6453f80-87f4-11eb-91ad-e31c446ce186.png)

When I build the previous commit (b8f1156315e656ee7855498d988ff7cfaff98d9d) everything builds fine. I use the following Dockerfile do the build:

```bashFROM ros:melodic
RUN apt update -qq && \
    apt install -y --no-install-recommends \
    software-properties-common

RUN add-apt-repository ppa:joseluisblancoc/mrpt && \
    add-apt-repository ppa:joseluisblancoc/gtsam-develop

RUN apt update -qq && \
    apt install -y --no-install-recommends \
    curl \
    git \
    cmake \
    libmrpt-dev \
    mrpt-apps \
    libgtsam-dev \
    libboost-serialization-dev \
    libboost-system-dev \
    libboost-filesystem-dev \
    libboost-thread-dev \
    libboost-program-options-dev \
    libboost-date-time-dev \
    libboost-timer-dev \
    libboost-chrono-dev \
    libboost-regex-dev

RUN mkdir -p /root/src

WORKDIR /root/src/

RUN git clone https://github.com/MOLAorg/mola.git

WORKDIR /root/src/mola/

# RUN git checkout b8f1156315e656ee7855498d988ff7cfaff98d9d

# RUN git pull

RUN git submodule update --init

RUN mkdir -p /root/src/mola/build

WORKDIR /root/src/mola/build/

RUN cmake ..

RUN cmake --build .
```
Is there something wrong in my configuration?

**Comment by jlblancoc** [comment]
Hi!

Sorry, the super repo wasn't updated correctly after the latest changes in a WIP module. 

It's fixed now after 7e9bb52.

**Comment by TankPi** [comment]
Thanks!
